core/src/main/scala/net/ruippeixotog/scalascraper/browser/HtmlUnitBrowser.scala

```diff
@@ -1,23 +1,20 @@
 package net.ruippeixotog.scalascraper.browser
 
-import java.io.{File, InputStream}
-import java.net.URL
-import java.nio.charset.Charset
-import java.util.UUID
-
-import scala.collection.JavaConverters._
-
+import net.ruippeixotog.scalascraper.browser.HtmlUnitBrowser._
```

Comment:
There is a significant change in import order that's making the diff harder to read. The convention I've been applying here was to have java.* and scala.* imports first, followed by 3rd party libs and local imports at the end.

I usually configure scalafix to enforce these rules but it looks like scala-scraper didn't have it configured, that was my bad. I have just done it in fbef0482b829cff9b09052437a42a80ef31ddf03, so if you merge/rebase with the latest master you can just run compile and imports should be organized/simplified automatically :)

Reply:
I have merged latest master commits, it should be resolved now.